buildbot-host/scripts/install-openvpn-build-deps-arch.sh

```diff
@@ -4,7 +4,7 @@
 #
 set -ex
 
-yes|pacman -Syu
+yes|pacman -Syu --noconfirm
```

Comment:
In this context I think this is the proper solution. Apparently (see [discussion](https://unix.stackexchange.com/questions/52277/pacman-option-to-assume-yes-to-every-question)) --noconfirm can break things, but that applies mainly to persistent systems. In case of Docker we don't need to worry about those changes imho.